patches/master_patch.patch

```diff
@@ -512,6 +571,113 @@ index d85ddf9db90d332a31f839e7600b8d6e5ec61356..2dbe745b19001ce55631e625b6c597b2
    // Finds TabStripModel which has a WebContents whose id is the given
    // |tab_id|, and returns the WebContents index and the TabStripModel.
    int FindTabStripModelById(int32_t tab_id, TabStripModel** model) const;
+diff --git a/chrome/browser/safe_browsing/chrome_password_protection_service.cc b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
+index 4275342f9244aa005521882c23767662fa2a681c..fc68b078ae1438a66e0bd7fe76b847cae8db9f2c 100644
+--- a/chrome/browser/safe_browsing/chrome_password_protection_service.cc
++++ b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
+@@ -239,13 +239,6 @@ void ChromePasswordProtectionService::ShowModalWarning(
+     web_contents->ExitFullscreen(true);
+ 
+   UpdateSecurityState(SB_THREAT_TYPE_PASSWORD_REUSE, web_contents);
+-  ShowPasswordReuseModalWarningDialog(
```

Comment:
we need to reduce the size of this patch. Can you provide some more info about the reasons for the patches?